docs/notebooks/basic-scrna-tutorial.ipynb

```diff
@@ -0,0 +1,1193 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    adata.obs["manual_anno"] = [new_cluster_names[x] for x in adata.obs["leiden_res0_5"]]
</pre><p> </p><p> </p><pre class="ql-syntax" spellcheck="false">adata.obs["manual_anno"] = adata.obs["leiden_res0_5"].map(new_cluster_names)
</pre><p>would be the pandas way of doing this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>